webrtc.html

```diff
@@ -3819,7 +3820,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       SHA-256 hash.</p>
       <div>
         <pre class="idl">partial interface RTCPeerConnection {
-    static Promise&lt;RTCCertificate&gt; generateCertificate (RTCGenerateCertificateOptions keygenAlgorithm);
+    static Promise&lt;RTCCertificate&gt; generateCertificate (AlgorithmIdentifier keygenAlgorithm);
```

Comment:
I don't get this. You're saying here that the argument is an AlgorithmIdentifier, and you're saying below that when generateCertificate is called with an Object, you will attempt to turn it into an RTCCertificateExpiration.
Is it an Object or an AlgorithmIdentifier?